providers/bind/soa.go

```diff
@@ -19,10 +24,18 @@ func makeSoa(origin string, defSoa *SoaDefaults, existing, desired *models.Recor
 		desired = &models.RecordConfig{}
 	}
 
+	soaMail := firstNonNull(desired.SoaMbox, existing.SoaMbox, defSoa.Mbox, "DEFAULT_NOT_SET.")
+	if strings.Contains(soaMail, "@") {
+		soaMail = soautil.RFC5322MailToBind(soaMail)
+	} else {
+		fmt.Println("WARNING: SOA hostmaster address must be in the format hostmaster@example.com")
+		fmt.Println("WARNING: hostmaster.example.com is deprecated and will be dropped in a future version")
```

Comment:
Please document this in documentation/functions/domain/SOA.md

Something like: NOTE: The email address should be specified like a normal RFC822/RFC5322 address (`user@hostname.com`). It will be converted into the BIND format (`user.hostname.com`) internally.  This has the benefit of being more human-readable plus DNSControl can properly handle escaping and other issues.

Reply:
@tlimoncelli done